pkgs/top-level/ocaml-packages.nix

```diff
@@ -1663,6 +1663,8 @@ let
 
         pcap-format = callPackage ../development/ocaml-modules/pcap-format { };
 
+        pds = callPackage ../development/ocaml-modules/pds { };
```

Comment:
This does not belong to `ocamlPackages`. Please move it to `pkgs/by-name/`.

Reply:
@vbgl @balsoft I'm going with `ocamlPackages` because it's also a dependency of another `opam` package called [hll](https://opam.ocaml.org/packages/hll/).

Reply:
That’s unrelated. This package provides a single binary whose behavior does not depend on the version of OCaml that is used for compiling it. It does not need to be included in the fixed-point computation of ocamlPackages and therefore should be defined outside of it.

Reply:
> That’s unrelated. This package provides a single binary whose behavior does not depend on the version of OCaml that is used for compiling it. It does not need to be included in the fixed-point computation of ocamlPackages and therefore should be defined outside of it.

Ah, I see. Dune also provides libraries, which do depend on the ocaml version. Agreed on moving out of `ocamlPackages` then.

Reply:
@balsoft @vbgl I see, let me rework this PR then and move to `pkgs/by-name/`.

Reply:
moved to `/by-name`.